release-notes.md

```diff
@@ -12,6 +12,13 @@ deployment/base/scripts/init_data/01.Partner.template.ini
 ### Deployment Scripts ###
     php deployment/updates/scripts/add_permissions/2022_08_22_game_services_add_partner.php
 
+## Add permission to insert epAllowedTemplates property on partner object for self serve partner ##
```

Comment:
Add permission to insert and update eventPlatformAllowedTemplates property on partner object